src/Builder/BuilderInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+
+namespace Prometee\SyliusPayumMoneticoPlugin\Builder;
+
+
+interface BuilderInterface
+{
+    /**
+     * @param $payload
+     * @return mixed
+     */
+    public function build($payload);
```

Comment:
Add an array return type explained string[] in the PHPDoc
Add a interface type (eg: `Sylius/*/Core/*Interface`) on the argument, rename it to `$payment` or `$address` and a PHPDoc

I will add easy-coding-standard after merging you PR to add some basic syntax check on this old plugin ;)